src/escape_chars.c

```diff
@@ -29,6 +29,9 @@ SEXP C_escape_chars_one(SEXP x) {
       case '\b':
       case '\f':
         matches++;
+        break;
+      case '/':
+        matches += escape_solidus;
```

Comment:
because it's optional, so it should only be incremented if `escape_solidus` is actually true.